themes/default/content/blog/pulumi-security-posture/index.md

```diff
@@ -0,0 +1,96 @@
+
+## Build Pulumi Teams
+
+[Teams](https://www.pulumi.com/docs/intro/pulumi-service/teams/#creating-a-team) improve on basic Role Based Access Control for Administrators and Users by allowing you to group users and their access to specific Stacks in your Organization.  For example, a team called ProductionReadOnly could be created and scoped so that all Members have only limited access to read the Production Stacks, while another team, ProductionAdmins, would retain full privileges to those stacks.
+
+## Write the Code
+
+Manual process is error prone and difficult to audit, iterate and improve on.  If you’ve created manual resources, consider [importing](https://www.pulumi.com/blog/changes-to-import/) them to Pulumi programs.
+
+## Tags your Stacks
+
+We have previously covered [Tag Policies](https://www.pulumi.com/blog/automatically-enforcing-aws-resource-tagging-policies/) as a best practice for the Cloud Resources themselves, but you can also tag your Pulumi Stacks to help organize and visualize your application resources in logical groupings that will help you assess, respond and automate incident remediation. Here are two examples of using Pulumi’s own Pulumi Provider to tag your Stacks automatically: in [python](https://github.com/pulumi/examples/blob/master/aws-py-stackreference/team/__main__.py#L8-L13) and in [typescript](https://github.com/pulumi/examples/blob/master/aws-ts-stackreference/team/index.ts#L17-L22).
+
+## Deploy from Pipelines
+
+Automate every step of your typical deployment process for anything beyond basic local development from a CI/CD pipeline or triggered by web services. If a human is in the loop, consider their involvement to be limited to the approvals process if needed. Better still is to proactively launch approved resources with Pulumi Multi Language Components and CrossGuard.
```

Comment:
- `Pulumi Multi Language Components`: I picked up somewhere (but can't find it anymore where) that the preferred naming is `Pulumi Packages`
- `CrossGuard`: add a link to our website, given it is the first occurrence to it.

Reply:
1. Updated Pulumi Multi Language Components with Pulumi Packages and hyperlinked it
2. Added the link to cross guard